forward.c

```diff
@@ -586,11 +594,15 @@ int forward_tunnel(void *thread_data) {
 	assert(thread_data != NULL);
 	int cd = ((struct thread_arg_s *)thread_data)->fd;
 	char *thost = ((struct thread_arg_s *)thread_data)->target;
+	char *thost_noport = strdup(thost);
```

Comment:
I'd rather call this `hostname`

Reply:
Also, after `bailout:` (near "free(tcreds)") you must add a `free(thost_noport)` (or "free(hostname)" if you change the name)

Reply:
Yep, added and renamed as you suggested!!